src/MultiSelectField.js

```diff
@@ -1,83 +1,95 @@
 import propTypes from '@dhis2/prop-types'
 import React from 'react'
 
-import { statusPropType } from './common-prop-types.js'
+import { statusPropType, multiSelectedPropType } from './common-prop-types.js'
 
 import { Field } from './Field.js'
 import { Label } from './Label.js'
 import { Help } from './Help.js'
-import { Select } from './Select.js'
+import { MultiSelect } from './MultiSelect.js'
 
 /**
  * @module
  *
- * @param {SelectField.PropTypes} props
+ * @param {MultiSelectField.PropTypes} props
  * @returns {React.Component}
  *
- * @example import { SelectField } from '@dhis2/ui-core'
+ * @example import { MultiSelectField } from '@dhis2/ui-core'
  *
  * @see Specification: {@link https://github.com/dhis2/design-system/blob/master/molecules/select.md|Design system}
- * @see Live demo: {@link /demo/?path=/story/select-regular--no-value|Storybook}
+ * @see Live demo: {@link /demo/?path=/story/multiselectfield--default|Storybook}
  */
-class SelectField extends React.Component {
+class MultiSelectField extends React.Component {
     render() {
         const {
             className,
             onChange,
             onFocus,
             onBlur,
-            initialFocus,
-            dense,
             required,
             label,
-            disabled,
-            name,
             valid,
             error,
+            disabled,
             warning,
             loading,
-            value,
+            selected,
             tabIndex,
             helpText,
             validationText,
+            maxHeight,
             children,
+            clearable,
+            clearText,
+            filterable,
+            filterPlaceholder,
+            placeholder,
+            prefix,
+            empty,
+            loadingText,
+            noMatchText,
+            initialFocus,
+            dense,
         } = this.props
 
         return (
             <Field className={className}>
-                {label ? (
-                    <Label
-                        required={required}
-                        disabled={disabled}
-                        htmlFor={name}
-                    >
-                        {label}
-                    </Label>
-                ) : null}
+                {label && <Label required={required}>{label}</Label>}
```

Comment:
Why not passing the `disabled` and `htmlFor` props to the label?
Clicking on the label should do the same as what `initialFocus` does right now

Reply:
I'm not passing htmlFor because it's not valid if it's not pointing to a labelable element (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label). We're not using an actual input element, so it's not valid. So having a click on label focus the input also won't work, at least not if we rely on native behaviour.

I missed passing the disabled prop to label, good catch. Fixed that in 2105488e26b67439c059ac1a7f43d647b90381d0

Reply:
I think it'd be weird if all our fields support clicking the label but this one doesn't.
I don't know how to solve this right now, one idea could be:
Make the focus state of the `Select` controllable:
1. Use `componentDidUpdate` to check the focus prop
1. Use an `onBlur` event handler (which I think it should have anyways, like all other components)
1. The `SingleSelect` and `MultiSelect` just pass through these props
1. The `...Fields` have an actual state and hook into the `onBlur` prop to know when to reset the value

Reply:
The Select has an onBlur event handler and the focus state is already controllable btw. But since that happens at the level of the select, we'd have to lift up the definition of the focusing logic to be able to manage it from the `*Select` and `*SelectField`.

That introduces some interdependency that might be a bit confusing. I think this is complex enough to warrant handling it in a follow-up PR.

Reply:
I'll add it to the todo list.